development/lib/run-command.js

```diff
@@ -87,17 +88,20 @@ async function runCommand(command, args) {
  *
  * @param {string} command - The command to run
  * @param {Array<string>} [args] - The arguments to pass to the command
+ * @param {string} output - The output file to write to
  */
-async function runInShell(command, args) {
+async function runInShell(command, args, output) {
   let errorSignal;
   let errorCode;
   const internalError = new Error('Internal');
   try {
     await new Promise((resolve, reject) => {
-      const childProcess = spawn(command, args, {
-        encoding: 'utf8',
-        stdio: 'inherit',
-      });
+      const childProcess = spawn(command, args);
+      childProcess.stdout.setEncoding('utf8');
+      childProcess.stdout.pipe(process.stdout);
+      if (output) {
+        childProcess.stdout.pipe(fs.createWriteStream(output));
+      }
```

Comment:
Here we pipe the xunit output to the console
Then we create an XML file from the output